### ISSUE TITLE
[req]: Filter torrent by file size

### ISSUE THREAD

**Issue by Kroissan** [Jackett/Jackett#12854]
### Is there already a request for your feature?

- [X] I have checked older issues, open and closed

### Is your feature request related to a problem? Please describe.

I'm using jackett along qbittorrent rss downloader, mainly to farm ratio. But i don't want to grab 50gb bluray remux when doing so.
It would be a usefull feature to be able to filter torrent by size.

### Describe the solution you'd like

https://iptorrents.com/t.rss?u=123456;s0=20;s1=50

**Comment by ilike2burnthing** [comment]
Unfortunately this doesn't translate across to the site's search:
https://iptorrents.com/t?s0=10;s1=50;q=spooks or
https://iptorrents.com/t?s0=10&s1=50&q=spooks

Here they aren't size limits but rather seeders limits. If you can find out what the equivalent for size is, if there is one, then this could be implemented in the indexer config.

**Comment by Kroissan** [comment]
what i meant is actually if it would be possible for you guys to include the same filter on jackett api. We would do it like so:

**Comment by ilike2burnthing** [comment]
Ah, ok, got you now.

@ngosang would it be possible to add min & max size filters to the API?

**Comment by ngosang** [comment]
> would it be possible to add min & max size filters to the API?

Yes, but it will be a "post-filter", just like limit. We search as usual without the size filter and we apply the filter just before returning the results. We will return less than 100 results but I think it's acceptable. We can add the filter in the Jackett UI too.
